go.mod

```diff
@@ -93,6 +93,7 @@ require (
 	github.com/authgear/oauthrelyingparty v1.5.0
 	github.com/coder/websocket v1.8.12
 	github.com/go-ldap/ldap/v3 v3.4.8
+	github.com/rivo/uniseg v0.4.7
```

Comment:
Need to update e2e/go.mod e2e/go.sum custombuild/go.mod custombuild/go.sum as well

Reply:
Fixed in later commit